File: src/main/java/com/jss/osiris/modules/quotation/service/SimpleProvisionService.java
package com.jss.osiris.modules.quotation.service;

import com.jss.osiris.modules.quotation.model.SimpleProvision;

public interface SimpleProvisionService {
    public SimpleProvision getSimpleProvision(Integer id);

    public SimpleProvision addOrUpdateSimpleProvision(SimpleProvision simpleProvision);
}

File: client/src/app/modules/quotation/services/simple.provision.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { AppRestService } from 'src/app/services/appRest.service';
import { SimpleProvision } from '../../quotation/model/SimpleProvision';

@Injectable({
  providedIn: 'root'
})
export class SimpleProvisionService extends AppRestService<SimpleProvision>{

  constructor(http: HttpClient) {
    super(http, "quotation");
  }

}
